content/setting-up-tugboat/select-repo-settings.md

```diff
@@ -125,6 +126,12 @@ credentials from within the repo settings.
 
 ![Authenticate with a Docker registry](../../_images/authenticate-with-a-docker-registry-add-credentials.png)
 
+### Preview IP Filters
```

Comment:
This is about to be an Enterprise-only feature. Do we need to call that out somehow? We don't really have a precedent set for documenting features available at different tiers.

Reply:
Yes. For now, I'm thinking about using a shortcode to get one of the colored boxes, and call it out as Enterprise-only, ala: https://learn.netlify.app/en/shortcodes/notice/

Maybe a new one that we only use to denote Enterprise features?

Or if we think we may ever have other tiers and want to denote this is a "tier-specific" feature and not just default to "Enterprise," there may be benefit in using more general language in the header part. I'll ponder some options and maybe we can do an after-talk on the scrum on Wednesday.

Reply:
Regarding this, #213 creates a custom "notice" style so we can add a callout to denote this as an enterprise-only feature.

I'll also add something to the marketing site for this once we get the docs all set up.

Reply:
I like the call-out in #213